CliFx.Tests/HelpTextSpecs.Commands.cs

```diff
@@ -110,5 +114,121 @@ private class EnvironmentVariableCommand : ICommand
 
             public ValueTask ExecuteAsync(IConsole console) => default;
         }
+
+        [Command("cmd-with-defaults")]
+        private class DefaultArgumentsCommand : ICommand
+        {
+            // Standardize accross a culture on our tests so our unit tests are consistent
+            // regardless of the machine they're being run on.
+            public static CultureInfo Culture { get; } = CultureInfo.CurrentCulture;
+
+            [CommandOption(nameof(Object))]
+            public object? Object { get; set; } = 42;
+
+            [CommandOption(nameof(String))]
+            public string? String { get; set; } = "foo";
+
+            [CommandOption(nameof(EmptyString))]
+            public string EmptyString { get; set; } = "";
+
+            [CommandOption(nameof(WhiteSpaceString))]
+            public string WhiteSpaceString { get; set; } = " ";
+
+            [CommandOption(nameof(Bool))]
+            public bool Bool { get; set; } = true;
+
+            [CommandOption(nameof(Char))]
+            public char Char { get; set; } = 't';
+
+            [CommandOption(nameof(Sbyte))]
+            public sbyte Sbyte { get; set; } = -0b11;
+
+            [CommandOption(nameof(Byte))]
+            public byte Byte { get; set; } = 0b11;
+
+            [CommandOption(nameof(Short))]
+            public short Short { get; set; } = -1234;
+
+            [CommandOption(nameof(Ushort))]
+            public short Ushort { get; set; } = 1234;
+
+            [CommandOption(nameof(Int))]
+            public int Int { get; set; } = 1337;
+
+            [CommandOption(nameof(Uint))]
+            public uint Uint { get; set; } = 2345;
+
+            [CommandOption(nameof(Long))]
+            public long Long { get; set; } = -1234567;
+
+            [CommandOption(nameof(Ulong))]
+            public ulong Ulong { get; set; } = 12345678;
+
+            [CommandOption(nameof(Float))]
+            public float Float { get; set; } = 123.4567F;
+
+            [CommandOption(nameof(Double))]
+            public double Double { get; set; } = 420.1337;
+
+            [CommandOption(nameof(Decimal))]
+            public decimal Decimal { get; set; } = 1337.420M;
+
+            // Standardizing on the US culture for the test so that our unit tests are consistent across cultures.
+            [CommandOption(nameof(DateTime))]
+            public DateTime DateTime { get; set; } = DateTime.Parse("Apr 20, 2020", Culture);
```

Comment:
Instead of using `Parse` you can just create a new object with `new` so that way you don't have to worry about culture.

Reply:
Yeah you're totally right. Fixed.